packages/tupaia-web/src/features/PDFExportDashboardItem/PDFExportHeader.tsx

```diff
@@ -5,40 +5,42 @@
 
 import React, { ReactNode } from 'react';
 import styled from 'styled-components';
-import { FlexCenter as BaseFlexCenter, FlexColumn as BaseFlexColumn } from '@tupaia/ui-components';
 
-const FlexCenter = styled(BaseFlexCenter)`
-  position: relative;
-  padding: 50px;
+const Container = styled.div`
+  align-items: center;
+  display: grid;
+  gap: 5mm;
+  grid-template-columns: 1fr 2fr 1fr;
+  width: 100%;
 `;
 
-const FlexColumn = styled(BaseFlexColumn)`
-  text-align: center;
+const HeaderImage = styled.img`
+  max-height: 3cm;
+  max-width: 3.5cm;
 `;
 
 const Heading = styled.h1`
+  font-size: 1.625rem;
   font-weight: ${({ theme }) => theme.typography.fontWeightBold};
+  inline-size: 100%;
   line-height: 1.4;
-  font-size: 1.625rem;
-  text-transform: capitalize;
   margin: 0;
+  text-align: center;
 `;
 
-const Logo = styled.img.attrs({
-  src: '/tupaia-logo-dark.svg',
-})`
-  top: 1.875rem;
-  left: 1.2rem;
-  position: absolute;
-`;
-
-export const PDFExportHeader = ({ children }: { children: ReactNode }) => {
+export const PDFExportHeader = ({
+  imageUrl = '/tupaia-logo-dark.svg',
+  imageDescription = 'Tupaia logotype',
+  children,
+}: {
+  imageUrl?: string;
+  imageDescription?: string;
+  children: ReactNode;
```

Comment:
Can we make a separate interface above for this now that there are several props? My usual rule of thumb is 2 or more props = interface (although there may be some floating around that don't adhere to this rule, but since we are in here we may as well do it for this file)